Makefile

```diff
@@ -12,33 +12,20 @@ all_artifacts:
 	open ./Products
 	echo "🏁 Ended at $$(date +%T)"
 
-frameworks: clean
-	echo "👉 Creating dynamic libraries. Will take a while... Logs available: DerivedData/fastlane.log"
-	bundle exec fastlane build_xcframeworks > DerivedData/fastlane.log
-	echo "👉 Creating compressed archives"
-	cp ./LICENSE ./Products/LICENSE
-	make zip_artifacts name="StreamChat-All" pattern=./*.xcframework
-	make zip_artifacts name="StreamChat" pattern="./StreamChat.xcframework ./LICENSE"
-	make zip_artifacts name="StreamChatUI" pattern="./StreamChatUI.xcframework ./LICENSE"
-	make swiftpm_checksum
-	make clean
+frameworks:
+	bundle exec fastlane clean_products
+	echo "👉 Creating dynamic libraries. Will take a while... Logs available: fastlane/fastlane.log"
+	bundle exec fastlane build_xcframeworks > fastlane/fastlane.log
+	bundle exec fastlane compress_frameworks
+	bundle exec fastlane clean_products
 
-static_libraries: clean
```

Comment:
What about keeping the preconditions on clean?

Reply:
Returned this 👍